components/EditProfilePage/ProfileSettingsModal.tsx

```diff
@@ -132,65 +88,43 @@ export default function ProfileSettingsModal({
       <Modal.Header closeButton>
         <Modal.Title id="notifications-modal">{t("setting")}</Modal.Title>
       </Modal.Header>
-      <StyledModalBody>
+      <Modal.Body className={`p-3`}>
         <Form>
-          <div
-            /* remove "div w/ d-none" for testing and/or after Soft Launch 
-               when we're ready to show Email related element to users
-            */
-            className="d-none"
+          <HideableSection hideIf={!flags().notifications}>
+            {/* awaiting the notifications feature to come online */}
+            <ModalSubheader title={t("forms.notification")} />
+            <Row className={`p-2 d-flex justify-content-md-end`}>
+              <Col className={`col-12 col-md-8 mb-3 mb-md-0`}>
+                <small>{t("forms.notificationText")}</small>
+              </Col>
+              <Col className={`col-fill`}>
+                <ToggleButton
+                  size={"sm"}
+                  toggleState={notifications === "None"}
+                  stateTrueLabel={t("enable")}
```

Comment:
I am unable to toggle this button to switch it to the "enable" state; it's always "enabled" for me